admin_guide/high_availability.adoc

```diff
@@ -14,34 +14,37 @@ toc::[]
-of replicas, are satisfied when the appropriate resources are available. When
-run with two or more replicas, the
-xref:../architecture/core_concepts/routes.adoc#routers[router] can be resilient
-to failures, providing a highly-available service. Depending on how the router
-instances are discovered (via a service, DNS entry, or IP addresses), this could
-impose operational requirements to handle failure cases when one or more router
-instances are "unreachable".
-
-For some IP-based traffic services, virtual IP addresses (VIPs) should always be
-serviced for as long as a single instance is available. This simplifies the
-operational overhead and handles failure cases gracefully.
-
-IMPORTANT: Even though
-a service is highly available, performance can still be affected.
+IP Failover (IPF) provides high availibilty for a service in the cluster.
+IPF uses the link:http://www.keepalived.org/[*Keepalived*] daemon running in pods
+in the cluster. The admin can select the nodes that will participate or use all
+nodes in the cluster.  The admin provides a pool of one or more externally
+accessible IP addresses (VIP) that will be served. *Keepalived* makes sure that each
+VIP is served on only one of the nodes at any point in time.
+
+When using VIPs to access a service, such as a router, the service must be running on all of
+the nodes that are running the IP Failover (IPF) pods. *Keepalived* monitors a port
+in the service to determine that the service it is running. As part of normal operation
+*keepalived* may assign the VIP to any of the nodes at any point in time.
+
+The nodes in the high availibility set can come and go. The service on the node can start
+and stop. The VIPs will be serviced from one of the remaining nodes and it will be
+serviced as long as at least one node with a running service is available.
+
+IMPORTANT: Even though a service is highly available, performance can still be affected.
 
 Use cases for high-availability include:
 
-* I want my cluster to be assigned a resource set and I want the cluster to automatically manage those resources.
-* I want my cluster to be assigned a set of VIPs that the cluster manages and migrates (with zero or minimal downtime) on failure conditions, and I should not be required to perform any manual interactions to update the upstream "discovery" sources (e.g., DNS). The cluster should service all the assigned VIPs when at least a single node is available, despite the current available resources not being sufficient to reach the desired state.
+* I want external users to have a single IP address to access my service which may be running on multiple nodes.
+* I care that when the node that is servicing the IP address goes down another node is automatically chosen to service the IP address.
+* I don't want to manually update the upstream network configuration (e.g., DNS) when nodes change.
+* I want the service to be available as long as at least one node is available.
```

Comment:
This feels wrong.  The service proxy already provides that.  Failover is entirely about external IP addresses.